### PR TITLE
LPS-168299 change title object field name to display date correctly

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
Related issue: https://issues.liferay.com/browse/LPS-168299

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFormProvider.java [hunk 1]
```diff
@@ -515,7 +516,19 @@ private String _getRelationshipLabelFieldName(ObjectField objectField) {
...
+			return StringUtil.replace(
+				titleObjectFieldName, "createDate", "dateCreated");
+		}
+
+		if (titleObjectFieldName.equals("modifiedDate")) {
+			return StringUtil.replace(
+				titleObjectFieldName, "modifiedDate", "dateModified");
+		}
+
+		return titleObjectFieldName;
```
Comment (carolmariaabb) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFormProvider.java:531]: @pedro-oliveira446 great job!

Could you please check if we can simplify to:

```
if (titleObjectField == null) {
	return "id";
}
else if (Objects.equals(titleObjectField.getName(), "createDate")) {
	return "dateCreated";
}
else if (Objects.equals(titleObjectField.getName(), "modifiedDate")) {
	return "dateModified";
}

return titleObjectField.getName();
```

Thanks!

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFormProvider.java (hunk 1 above)
Comment (pedro-oliveira446) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFormProvider.java]: @carolmariaabb Apparently it's working properly.

Thanks.

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
Manually forwarded https://github.com/brianchandotcom/liferay-portal/pull/126523

**Pull request closed without merging**